invenio_notify/tasks.py

```diff
@@ -237,7 +271,7 @@ def create_endorsement_record(identity, record_item: Union[str, RDMRecordMetadat
         create_endorsement_update_notification(
             record_id,
             actor_name,
-            actor_type,
+            constants.WORKFLOW_STATUS_ANNOUNCE_REVIEW,
```

Comment:
no related to this, PR
but better don't call it actor_type
that logical incorrect 
`actor_type == constants.TYPE_REVIEW`